docs/source/guides/tools/rpakpilotmodding.rst

```diff
@@ -0,0 +1,245 @@
+Delete the pilot models head and body after doing so. 
+Renaming and putting your model in the correct collection.
+
+- Rig the model to match the bones of the armature that came with the pilot model
+- Create an armature modifier for your model’s body and head
+
+.. image:: /_static/importpilotmodel/ss0-pilotBlenderArmature.png
+   :align: center
+   :class: screenshot
+
+
+Attach the model’s body and head to the armature.
+The video does not have the model properly aligned with the armature. Please make sure your model is aligned with the armature before attaching the model to it.
+A properly rigged model should look like this:
+
+.. image:: /_static/importpilotmodel/ss1-pilotBlender.png
+   :align: center
+   :class: screenshot
+
+If you have multiple textures on your model, you would create several materials that match the name of the textures for the pilot you are modifying.
+If you only have one texture for your model, create one material for your entire model.
+Use LegionPlus to view paths for materials. Scroll below a little bit to learn how to use Legion+.
+
+.. image:: /_static/importpilotmodel/ss2-pilotBlenderMatl.png
+   :align: center
+   :class: screenshot
+
+Once you have made all the changes you wanted to, select ``Scene Properties``, ``Source Engine Export``, then set the export format to ``SMD`` and set your export path.
+
+.. image:: /_static/importpilotmodel/ss3-BlenderQcModifier.png
+   :align: center
+   :class: screenshot
+
+Copy the qc file to the same location where you exported the model.
+
+Modifying the POV model
+^^^^^
+
+- Select ``File`` in Blender, then select ``Save Copy``
+- Choose any name, save the copy, then open the copy
+- Delete everything except for the body, the body collection, and the skeleton
+- Add ``pov_`` to the beginning of your body model and armature
+- Repeat the process of exporting the model with ``Source Engine Export``
+
+Using Legion+ to view paths
+^^^^^
+
+Open `Legion+`_
+-----
+
+Select ``Load File``
+Follow the directory that matches your launcher.
+
+1. Steam: ``Steam\steamapps\common\Titanfall2\r2\paks\Win64\common.rpak`` or
+2. Origin: ``Origin Games\Titanfall2\r2\paks\Win64\common.rpak``
+
+Type the file name of the pilot into the search bar to view **texture and material paths**.
+View file paths to create paths in RePak
+
+Recompiling the model
+^^^^^
+
+Install compatible Source game
+----
+
+You need a compatible Source game installed when using Crowbar.
+Compatible Source games in order of preference:
+
+Compatible Source games
+#####
+
+1. Portal 2
+2. Alien Swarm
+3. Source FilmMaker
+
+Open `Crowbar`_
+-----
+
+Select ``Compile``
+Click on the Browse button next to ``QC input`` then locate and select your modified model.
+Next, Click on the Browse button next to ``Output to`` then choose a location to output the model to.
+
+Making a Pak file
+^^^^^
+
+Follow this `tutorial <https://r2northstar.readthedocs.io/en/latest/guides/rpakmodding.html>`_ to create a pak file
```

Comment:
```suggestion
Making an RPak file
^^^^^

Follow this `tutorial <https://r2northstar.readthedocs.io/en/latest/guides/rpakmodding.html>`_ to create a RPak file
```
Just for consistency, RPak shouldn't be shortened to just Pak (some people might get confused)

Reply:
Didn't notice. Will use this commit.